docs/DeveloperGuide.md

```diff
@@ -248,47 +248,61 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | Product Manager                          | mark deliverables as completed                                              | refer back to them when needed     |
 | `* * *`  | Product Manager                          | delete deliverables that are no longer relevant                             | focus on current deliverables      |
 | `* *`    | Product Manager                          | tag deliverables as delayed and change the deadlines                        |                                    |
-| `* *`    | Product Manager                          | change the person-in-charge of deliverables                                 | locate a person easily             |
-| `* *`    | Product Manager                          | undo my action / be prompted before any action                              | Minimise mistakes made             |
-| `* *`    | Product Manager                          | view the overall completion percentage of the project                       | know whether we are on track (completed deliverables / total deliverables) |
+| `* *`    | Product Manager                          | change the person-in-charge of deliverables                                 | locate the right person easily     |
+| `* *`    | Product Manager                          | undo my action / be prompted before any action                              | minimise mistakes made             |
+| `* *`    | Product Manager                          | view the overall completion percentage of the project                       | know whether we are on track (e.g _`completed deliverables` out of `total deliverables`_) |
 | `* *`    | Product Manager                          | package deliverables into different iterations                              | retrieve the relevant deliverables easily |
 | `* *`    | Product Manager                          | easily know who and how to contact the person-in-charge                     | contact the person when the progress of a deliverable is behind schedule |
 | `* *`    | Product Manager                          | flag problematic deliverables in the app                                    | know that these tasks require further attention |
 | `* *`    | Product Manager                          | be notified when my product’s development is close to tagline / past deadline / too slow |                       |
 | `* *`    | Product Manager                          | pin different parts of the project according to their urgency               |                                    |
-| `*`      | Product Manager                          | see the previous products I have handled and their reviews                  |                                    |
-| **Contacts** |
+| `*`      | Product Manager                          | see the previous products I have handled and their reviews                  | remember and retrieve important information |
```

Comment:
I believe you added the 'so that' portion wrongly here. It should be for the "view my contacts and their relevant details" user story under EPIC B. Could you help to change that?